app/models/token.rb

```diff
@@ -50,7 +50,8 @@ def validate_token
   def valid_issuer?
     decoded_token = JWT.decode(@token_string, nil, false, algorithm: 'RS256')
     iss = decoded_token[0]['iss']
-    !iss.nil? && iss.match?(%r{^#{Regexp.escape(Settings.oidc.issuer_prefix)}/\w+$})
```

Comment:
This is going to change some behavior and permit unexpected issuers. Let's have a call to walk through when Derek is available.